sysutils/monit/Portfile

```diff
@@ -22,8 +22,8 @@ long_description    Monit can start a process if it does not run, restart a proc
 homepage            https://mmonit.com/monit/
 master_sites        ${homepage}dist/
 
-checksums           rmd160 a0e8aedcc73d36354b1362cb1e30e71ac9114f62 \
-                    sha256 f71a22cfb6bd91ff46496e72e1d1b1021ecd651e7748131ce0f995cc37ff0b42
+checksums           rmd160 5691b306930cfa8493c771bf79d061ddeefca440 \
+                    sha256 aa0ce6361d1155e43e30a86dcff00b2003d434f221c360981ced830275abc64a
```

Comment:
We generally now add a "size" line in the checksums section. See other ports if it isn't obvious how to add it. Note that if you just add "size 0" at the end of checksums and do "port checksum -v" it will spit out the right line for you.

Reply:
Fixed.

Thanks for the advice! I didn't get a size from that command, but it did validate when I had the correct value for `size`.